core/ir/encode_shared.c

```diff
@@ -99,6 +99,7 @@ get_encoding_info(instr_t *instr)
     const instr_info_t *info = instr_get_instr_info(instr);
     decode_info_t di;
     decode_info_init_for_instr(&di, instr);
+    IF_AARCHXX(di.check_reachable = false;)
```

Comment:
From the PR description it looks like this PR is only for ARM, but here it's changing for AArch64 too.

Reply:
Commenting as a review comment to make it visible on front PR page.

Reply:
Changing the description in the final merge.